alsa-mixer.cabal

```diff
@@ -18,10 +18,16 @@ Source-repository head
   Type:     git
   Location: https://github.com/ttuegel/alsa-mixer.git
 
+Flag cross
+  Description:         Set this flag if cross-compiling
+  Default:             False
+  Manual:              True
+
 Library
   Exposed-modules:     Sound.ALSA.Mixer
   Other-modules:       Sound.ALSA.Mixer.Internal
-  Build-tools:         c2hs
+  if !flag(cross)
```

Comment:
Shouldn't `Build-tools` for cross-compiling be handled by Cabal?

Reply:
Yes! But as of now it is not; see e.g. https://github.com/haskell/cabal/issues/5760